lighthouse-core/report/templates/report-template.html

```diff
@@ -247,8 +249,14 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
 {{/each}}
 
 <script>
-window.addEventListener('DOMContentLoaded', _ => {
-  new LighthouseReport(window.lhresults);
+(function(exports) {
```

Comment:
okay here's how report-generator is consumed:
![image](https://cloud.githubusercontent.com/assets/39191/22721743/638725a8-ed67-11e6-82e6-619770327321.png)

`viewer/app/src/main` and `lighthouse-background` are browserified and used in the viewer and extension/devtools, respectively.

question:

* why is report-generator including the source of `scripts/lighthouse-report` into the report when that file was already part of the bundle that shipped with the viewer

also, can we rename some of these files, because they all sound like the same thing right now. :)